### PR TITLE
FOGL-2420: Added mention of libcurl-dev dependency

### PR DESCRIPTION
**Pull request by AmandeepArora**

### CODE REVIEW

**Review by praveen-garg — APPROVED** [review]
lgtm!

@monika-Dianomic Please test this dependency installation (make deb) on Rpi

File: README.rst [hunk 1]
```diff
@@ -4,6 +4,15 @@ FogLAMP "email notification" plugin
...
 Email notification plugin
 
+Note:
+
+This plugin requires the installation of libcurl-dev apt package and that
+is a virtual package provided by 'libcurl4-openssl-dev' among other options.
+
+.. code-block:: console
+
+  $ sudo apt-get install libcurl4-openssl-dev
```
Comment (MonikaSharma06) [inline README.rst:14]: Got this error on Rpi:
`Err:1 http://raspbian.raspberrypi.org/raspbian stretch/main armhf libcurl4-openssl-dev armhf 7.52.1-5+deb9u8
  404  Not Found [IP: 5.153.225.207 80]
E: Failed to fetch http://raspbian.raspberrypi.org/raspbian/pool/main/c/curl/libcurl4-openssl-dev_7.52.1-5+deb9u8_armhf.deb  404  Not Found [IP: 5.153.225.207 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?`

Please fix it.

**Review by MonikaSharma06 — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (AmandeepArora) [reply README.rst:14]: I think other people outside are also getting this issue today:

https://www.raspberrypi.org/forums/viewtopic.php?t=232928&p=1425878

There is probably problem with some apt server. Even "apt upgrade" upgraded 2/3rd of the packages and failed for 1/3rd of them.

Comment (MonikaSharma06) [reply README.rst:14]: Yeah, that should be the reason. Approving for now as it works fine on ubuntu 16 and 18. Will verify it when the issue is resolved on Rpi.

**Review by MonikaSharma06 — APPROVED** [review]

**Pull request merged**